src/components/carousel/index.jsx

```diff
@@ -203,6 +264,8 @@ const Carousel = ({
 						{pageCountPhrase(slide, totalSlides) || `${slide} of ${totalSlides}`}
 					</p>
 				) : null}
+				{AutoplayEnabledAndAllowed && !isAutoplaying ? resolvedStartAutoplayButton : null}
```

Comment:
Using two buttons - one for each interaction will require us to manage state and focus for the user - could we get away with using one and have the user set the label and icon?

Reply:
I was following the same custom button pattern as the next and previous buttons. What if the user wants to put an icon on the left or right side of the label? or do something else custom? should I add an icon left or right option as well? 

new props: 
- autoplayStartLabelText
- autoplayStartIcon
- autoplayStartIconLocation 
- autoplayStopLabelText
- autoplayStopIcon
- autoplayStopIconLocation

Reply:
I'm not totally sure how I could avoid using state here. what would determine whether to show the autoplayStartLabelText vs autoplayStopLabelText ? is this an instance of using aria attributes? @matthewroach

Reply:
I don't think we need so many options - what if we provide the ability for them to set the label and icon for each state for now?

Reply:
Are you thinking of making the toggle button a callback where we pass in the current state and letting them decide?

Reply:
yep can do @matthewroach 

> I don't think we need so many options - what if we provide the ability for them to set the label and icon for each state for now?

Reply:
this has been updated @matthewroach